test/visual-diff/visual-tests

```diff
@@ -249,6 +249,20 @@
       "loading_complete_delay_ms": 500,
       "enable_percy_javascript": true
     },
+    {
+      "url": "examples/visual-tests/amp-story/amp-story-ad.html",
+      "name": "AMP Story Ad",
+      "viewport": {
+        "width": 400,
+        "height": 600
+      },
+      "interactive_tests": "examples/visual-tests/amp-story/amp-story-ad.js",
+       "loading_complete_selectors": [
+        "amp-story[ad-showing]",
+      ],
+      // for page navigation
+      "loading_complete_delay_ms": 500
```

Comment:
Do we know if this starts after "loading_complete_selector"?

Reply:
From the comments in the test file

 A duration of time (in milliseconds) to wait **after all
 other loading is complete,** before taking a snapshot. 
 
 In other words, yes, according to the docs, it will start after "loading_complete_selectors"
 and all other loading is done.

Reply:
thanks for confirming!